openapi.yml

```
@@ -145,6 +145,39 @@ paths:
           $ref: "#/components/responses/404NotFound"
         "409":
           $ref: "#/components/responses/409Conflict"
+  /organizations/{ipaCode}/delegate:
```

```suggestion
  /organizations/{ipaCode}/delegates:
```

use plural for resources